src/site/topics/bags/bag-implementations.rst

```diff
@@ -154,7 +154,7 @@ Remove
 
 
 * The ``remove(T element)`` method delegates to the ``remove(int index)`` for ease and code/logic reuse
-
+* Note that ``indexOf`` will throw an exception if the element does not exist within the bag 
```

Comment:
Doesn't it return -1?

Reply:
nah, there is a private `find` that returns the sentinel. `indexOf` throws an exception.